spark_pipeline_framework/utilities/spark_data_frame_helpers.py

```diff
@@ -51,11 +51,11 @@ def create_dataframe_from_dictionary(
     return df
 
 
-def create_empty_dataframe(spark_session: SparkSession) -> DataFrame:
-    schema = StructType([])
-
+def create_empty_dataframe(
+    spark_session: SparkSession, schema: Optional[StructType] = None
+) -> DataFrame:
```

Comment:
can you call isEmpty on the df returned from this function.

Reply:
is it returning True?

Reply:
Yes